modules/SpyUtils.js

```diff
@@ -42,6 +46,17 @@ export const createSpy = (fn, restore = noop) => {
     return returnValue
   }
 
+  if (supportsConfigurableFnLength) {
+    spy = Object.defineProperty(spyLogic, 'length',
+      { value: fn.length, writable: false, enumerable: false, configurable: true })
+  } else {
+    spy = new Function('spy', `return function(${ // eslint-disable-line no-new-func
+      Array(...Array(fn.length)).map((_, i) => Array(i + 2).join('_')).join(',')
```

Comment:
Good point. eslint complained about not allowing string concatenation so I replaced that with a template string (and replaced the `$` with a `_` to avoid this confusing expression \`$${i}`